docs/docs/cmd/teams/message/message-export.mdx

```diff
@@ -0,0 +1,170 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# teams message export
+
+Export Microsoft Teams chat messages for a given user or a team.
+
+## Usage
+
+```sh
+m365 teams message export [options]
+```
+
+## Options
```

Comment:
Lets do add some shorts here. Shorts may or may not be experienced as useful. But to stay consistent let's add them